//Created by wangjun 2021-10-19
// Test the grpc proto

syntax = "proto3";

package rpc_package;

// define a service
service MessageService {
    // define the interface and data type
    rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// define the data type of request
message HelloRequest {
    string name = 1;
}

// define the data type of response
message HelloReply {
    string message = 1;
}